sycl/test/on-device/sub_group/info.cpp

```diff
@@ -1,10 +1,8 @@
-// REQUIRES: opencl
+// XFAIL: cuda
```

Comment:
Do we have an issues regarding this failure. My expectations are that it should work on CUDA BE (there were set of fixes recently).

if the problem is in place please point to the issue in the comment.

Reply:
There's been no issue, so I created one: https://github.com/intel/llvm/issues/2922

Reply:
Could you please reference he issue in the source code comment?